src/Pages/Home.tsx

```diff
@@ -0,0 +1,157 @@
+import Header from '@/components/Header';
+import styled from '@emotion/styled';
+import { FiPlus } from 'react-icons/fi';
+import { mockThemeList } from '@/mocks/themeListMock';
+import ThemeItem from '@/components/ThemeItem';
+import type { ThemeItemType } from '@/types/theme';
+import RankingSection from '@/components/RankingSection/RankingSection';
+
+const Home = () => {
+  return (
+    <>
+      <Header title="선물하기" />
+      <HomeContainer>
+        <SelectFriendSection>
+          <SelectFriend>
+            <FiPlusIcon size={16} />
+            <SelectFriendText>선물할 친구를 선택해 주세요.</SelectFriendText>
+          </SelectFriend>
+        </SelectFriendSection>
+        <ThemeSection>
+          <ThemeSectionTitle>선물 테마</ThemeSectionTitle>
+          <ThemeGrid>
+            {mockThemeList.slice(0, 15).map((item: ThemeItemType) => (
+              <ThemeItem key={item.themeId} item={item} />
+            ))}
```

Comment:
```suggestion
            {mockThemeList.slice(0, 15).map((item: ThemeItemType) => (
              <ThemeItem key={item.themeId} {...item} />
            ))}
```
item 객체를 넘길게 아니라 item을 해체해서 넘겨주면 어떨까요!?
그리고 이게 렌더링 관점에서 어떤 차이가 있는걸까요!?